LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Should this (`BSD 3-Clause License`) appear in all the headers re: my pull request #35 ?

Reply:
Info on our license header requirements is at: https://sites.google.com/a/salesforce.com/open-source/open-source-project-guide/license-headers